static/deploy/python/infer.py

```diff
@@ -229,15 +232,17 @@ def __init__(self,
                  run_mode='fluid',
                  threshold=0.5,
                  trt_calib_mode=False,
-                 enable_mkldnn=False):
+                 enable_mkldnn=False,
+                 enable_mkldnn_bfloat16=False):
         super(DetectorSOLOv2, self).__init__(
             config=config,
             model_dir=model_dir,
             device=device,
             run_mode=run_mode,
             threshold=threshold,
             trt_calib_mode=trt_calib_mode,
-            enable_mkldn=enable_mkldnn)
+            enable_mkldn=enable_mkldnn,
+            enable_mkldnn_bfloat16=FLAGS.enable_mkldnn_bfloat16)
```

Comment:
Is the parameter directly from FLAGS `FLAGS.enable_mkldnn_bfloat16` used here on purpose? Shouldn't we use just the `enable_mkldnn_bfloat16` argument from the parent class defined above?